extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js

```diff
@@ -3604,7 +3654,7 @@ var globalRequire = require;
 						_amc.ui.adVideoElement.css(INVISIBLE_CSS);
 					}
 
-					if(_usingAdRules)
+					if(_usingAdRules && this.currentAMCAdPod.ad.forced_ad_type !== _amc.ADTYPE.NONLINEAR_OVERLAY)
 					{
 						_tryStartAd();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "_tryStartAd" before this usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)